basket/petition/views.py

```diff
@@ -16,6 +20,24 @@ def _add_cors(response):
     return response
 
 
+def _send_confirmation(request, petition):
+    pidb64 = urlsafe_base64_encode(str(petition.pk).encode())
+    confirm_link = request.build_absolute_uri(reverse("confirm-token", args=[pidb64, petition.token]))
+
+    email_subject = "Please confirm your open.mozilla.org/letter signature"
+    email_body = render_to_string(
+        "petition/confirmation_email.txt",
+        {
+            "name": petition.name,
+            "confirm_link": confirm_link,
+        },
+    )
+    email_from = "no-reply@mozilla.org"
```

Comment:
I think this is one of the approved from emails we can use. Did we confirm that? Last I used it SES required these be setup in their system.